### PR TITLE
Issuers updates & YouTube link addition

### PR DESCRIPTION
**Pull request by KreigD**
- Adds links to the "All Issuers" section
- Adds/updates issuers in top sections of the Issuers page
- Adds YouTube video link to home and FAQ pages

### CODE REVIEW

File: en/issuers.html [hunk 1]
```diff
@@ -193,7 +200,38 @@ <h1>
         <div class="row pt-4" id="issuers_list">
 
           {% comment %} Create a variable of the sorted list from the JSON file {% endcomment %}
-          {% assign issuers = site.data.vci-issuers | sort_natural: 'name' %}
+          {% assign issuers_grouped = site.data.vci-issuers | group_by: 'name' | sort_natural: 'name' %}
```
Comment (jmandel) [inline en/issuers.html]: This logic is getting a bit hard to follow. Is the issue that there are some entries with the same name, and we want to display each just once (including a website, if any)? That could be handled with something like:

    {% assign issuers = '' | split: '' %}
    {% assign issuers_grouped = site.data.vci-issuers | group_by: 'name' | sort_natural: 'name' %}
    {% for issuer_entry in issuers_grouped %}
        {% assign best_entry_for_issuer = issuer_entry.items | sort: "website" | last %}
        {% assign issuers = issuers | push: best_entry_for_issuer%}
    {% endfor }

**Review by jmandel — CHANGES_REQUESTED** [review]

File: en/issuers.html (hunk 1 above)
Comment (KreigD) [reply en/issuers.html]: Yeah, this is intended to filter out duplicates and prioritize any duplicate entries with a website value. I'll test out your proposed changes.

Comment (jmandel) [reply en/issuers.html]: Cool! (To be clear, I haven't tested the code above, so think of it as more like a sketch ;-)

Comment (KreigD) [reply en/issuers.html]: Looks like it works! Thanks, that's much more concise than what I came up with. I'll get that adjustment pushed here.
